examples/classic-typescript/package.json

```diff
@@ -16,8 +16,8 @@
     "dev": "docusaurus start"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.8",
-    "@docusaurus/preset-classic": "2.0.0-beta.8",
+    "@docusaurus/core": "../../packages/docusaurus",
```

Comment:
Please don't modify the example. Examples are generated after every release and are used in the playground.

Reply:
It was the only way to test the preset shorthand in the own repository. As it is validated, I will rollback this change then.

Reply:
> It was the only way to test the preset shorthand in the own repository. As it is validated, I will rollback this change then.

It's fine to modify the init templates (in `create-docusaurus`), and considering local testing, you can check out `CONTRIBUTING.md` for testing in your local environment. Usually (including this case) just testing on our own website is fine